package.json

```diff
@@ -6,7 +6,10 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.5.0",
```

Comment:
I think there is no need to add the @wojtekmai and enzyme as dependencies. I think we can just add them as devdependencies. At least that's what worked for me. I'm not sure if there is any difference really but maybe we can clarify with Xin.

Reply:
agree, when we install packages. if the package is only for dev purposes such as nodemon and all the testing lib (NOT as part of your applicatioin) they are installed with "npm install --save-dev nodemon" (short version npm i -D nodemon). otherwise "npm install react"

Reply:
Move these down to devDeps. It still not 100% clear what dependancies we need in the pipeline to test - assuming these are retested again (I thought we said we don't rerun units only regression in the pipeline). Can we discuss please?